src/moscot/problems/cross_modality/_mixins.py

```diff
@@ -183,6 +187,53 @@ def cell_transition(  # type: ignore[misc]
             key_added=key_added,
         )
 
+    def annotation_mapping(  # type: ignore[misc]
+        self: CrossModalityTranslationMixinProtocol[K, B],
+        mapping_mode: Literal["sum", "max"],
+        annotation_label: str,
+        forward: bool,
+        source: str = "src",
+        target: str = "tgt",
+        cell_transition_kwargs: Mapping[str, Any] = types.MappingProxyType({}),
+    ) -> pd.DataFrame:
+        """Transfer annotations between distributions.
+
+        This function transfers annotations (e.g. cell type labels) between distributions of cells.
+
+        Parameters
+        ----------
+        mapping_mode
+            How to decide which label to transfer. Valid options are:
+
+            - ``'max'`` - pick the label of the annotated cell with the highest matching probability.
+            - ``'sum'`` - aggregate the annotated cells by label then
+              pick the label with the highest total matching probability.
```

Comment:
```suggestion
              pick the label with the highest total matching probability.
              
```

Reply:
you might also have to indent further the bulletpoint, check readthedocs from this PR build how it looks like, if it looks wrong, then adapt by looking at how it's done in other places where bulletpoint are used in docstrings